### ISSUE TITLE
Cant Open BambuStudio on Windows 10

### ISSUE THREAD

**Issue by ADmakes** [bambulab/BambuStudio#72]
I used the installer and all I get is a splash screen saying "loading configuration", then white box , then it quits. Ive done several installs/uninstalls, restarts, and repaired the vcredist2019.
I have used the installer for both Versions xxx10.05 and xxx10.14 with the same results

**Comment by lanewei120** [comment]
Hello

Could you help to package the folder C:\Users\your_user_name\AppData\Roaming\BambuStudio\log into a zip
and upload it here to help us debug?

and also , could you help to show your windows version and your local language?

thanks

**Comment by ADmakes** [comment]
Ok here is the log file and screenshot of system info. Language is english


________________________________

**Comment by lanewei120** [comment]
I can not find your screenshot and log file,
there must be something missed

**Comment by ADmakes** [comment]
[AppData_log.zip](https://github.com/bambulab/BambuStudio/files/9182236/AppData_log.zip)
![Screenshot (31)](https://user-images.githubusercontent.com/67829954/180806340-236c7f32-df35-437c-812b-2ac47bd4bb80.png)

**Comment by lanewei120** [comment]
there is indeed a crash when init
however, we can not find the detailed stack in the formal release version

could you give an email here?
I will send your a package of the same version with pdb file(symbol table), 
then you can try again , 
and if it crashes, there will be a detailed stack which will be helpful

sorry for the inconvenience

**Comment by ADmakes** [comment]
adi_orio@hotmail.com

**Comment by ADmakes** [comment]
I have solved the problem!  
Windows media player needed activation. 
Although WMP was installed, it was not activated, and activating it was not obvious. 
In the SETTINGS>OPTIONAL FEATURES I had to click MORE WINDOWS FEATURES>Check the box for MEDIA FEATURES> then check WINDOWS MEDIA PLAYER, and it will install. Then Restart and Bambu Studio opens. Sharing this screenshot to help others. Thanks for your help
![Screenshot (34)](https://user-images.githubusercontent.com/67829954/180865741-c7fa75d0-1fc4-4c9b-beb2-28b5a7fde1f1.png)
